### PR TITLE
Add OS chooser

### PR DESCRIPTION
**Pull request by RDIL**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -103,10 +122,21 @@ def \
...
+        f.write("  osx_instance:\n")
+        f.write("    image: mojave-xcode-10.1\n")
+    elif win:
+        f.write("  windows_container:\n")
+        f.write("    image: cirrusci/windowsservercore:2019\n")
+        f.write("    os_version: 2019\n")
+    elif freebsd:
+        f.write("  freebsd_instance:\n"
+        f.write("    image: freebsd-11-2-release-amd64\n")
     keep = []
```
Comment (RDIL) [inline main.py]: @fkorotkov PTAL at this section whenever you have time

Comment (fkorotkov) [reply main.py]: LGTM. Are you planning to add tests to the generator? Something like mocking the inputs and checking the generated `.cirrus.yml`.

Comment (RDIL) [reply main.py]: Yeah, once PyCharm finishes updating I will add some.

Comment (RDIL) [reply main.py]: I’m going to make this project into a PyPI package so it is easier to test and distribute.

**Pull request merged**